openedx/core/lib/xblock_utils.py

```diff
@@ -396,3 +397,29 @@ def _course_info_content(html_parsed):
                         return payload
 
     return course_update_items
+
+
+def add_inline_analytics(user, block, view, frag, context):  # pylint: disable=unused-argument
```

Comment:
Well, it's referred to throughout Edx code as 'frag'. I'm liking the consistent approach unless you disagree violently.